quickstarts/install-odf.yaml

```diff
@@ -34,41 +34,40 @@ spec:
-      3. From your list of installed operators, click **OpenShift Container Storage.**
-      4. On the Storage Cluster tile, click **Create Instance.**
-      5. Select the cluster mode
-          - **Internal**: Can be used on any platform except bare metal
-          - **Internal-Attached device**: Can be used on any platform where there are attached devices to the nodes, using the Local Storage Operator (LSO)
-          - **External**: Used for external RHCS (Red Hat Cluster Suite) Cluster
-      6. Follow the on-screen instructions to create the storage cluster.
-      7. After creating the storage cluster, you'll be brought to the storage cluster details page. The state of the cluster appears beside the cluster name. Wait for the state of the cluster to change from Progressing to Ready. This may take a few minutes.
-      8. Once the state of the cluster is **Ready**, you can start using the OpenShift container storage resources (storage classes and the bucket classes) to allow OCS to manage your storage.
+      3. From your list of installed operators, click **OpenShift Data Foundation.**
+      4. On the Storage System tile, click **Create Instance.**
+      5. Select the backing storage
+          - **Use an existing storage class**
+          - **Create a new storage class using local storage devices:** Can be used on any platform where there are attached devices to the nodes, using the Local Storage Operator (LSO)
+          - **Create a new external storage class:** Used for connecting to an external storage platform to the ODF internal system
+      6. Follow the on-screen instructions to create the storage system.
+      7. Once the state of the storage system is **Ready**, you can start using the OpenShift container storage resources (storage classes and the bucket classes) to allow ODF to manage your storage.
     review:
       instructions: |-
-        #### Verify that the Storage Cluster is created:
+        #### Verify that the Storage System is created:
 
-        On the storage cluster details page, is the state of the cluster **Ready** ?
+        On the storage system details page, is the state of the system **Ready** ?
       failedTaskHelp: This task isn’t verified yet. Try the task again.
     summary:
-      success: Great work! You installed the OpenShift Container Storage operator.
+      success: Great work! You installed the OpenShift Data Foundation operator.
       failed: Try the steps again.
-  conclusion: Congratulations! The OpenShift Container Storage operator is ready to use. To learn how you can manage your storage space effectively, take the Getting Started With OpenShift Container Storage quick start.
+  conclusion: Congratulations! The OpenShift Data Foundation operator is ready to use. To learn how you can manage your storage space effectively, take the Getting Started With OpenShift Data Foundation quick start.
   nextQuickStart:
-    - "getting-started-ocs"
+    - "getting-started-odf"
```

Comment:
I don't see this quick start you are linking to.  Should it be part of this PR as well?

Reply:
Would it not be installed by the operator?

Reply:
@serenamarie125 It would be installed and managed by the odf operator.